xhttp/client_opts.go

```diff
@@ -51,7 +51,6 @@ func RetrierWithRespCheck() RetrierOption {
 }
 
 // RetrierWithSleep configures the sleep function used to sleep between retries, usually used for testing.
-// But can be used as a way to measure how much retries happened since this is called before each retry.
```

Comment:
This is irrelevant now that we have the callback functions, right?

Reply:
It is not irrelevant in the sense that it provides different information (the sleep times) and also it allows users to override sleeping behavior.

Reply:
ah if you mean the docs, yeah that part of the docs is misleading because we have better and more clear ways to do that now.

Reply:
Yeah, I just meant that one sentence in the docs.